### PR TITLE
Changes IDs

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
Generally adjusts IDs for all Skyrat-modified roles.

## Changelog

:cl:
Add: Blueshield CENTCOM to Silver ID
Add: NT Rep CENTCOM to Silver ID
Add: Weapon Permit access for QM
Add: Surgery access for Brigmed
Add: Representative Access for IDs and the office.
Add: Security access required for the Gateway.
del: CENT_GENERAL access required for the Gateway.
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
Small note, the Blueshield ID does not show up on a silver ID card for some reason. Gonna make an update soon to fix that.

**Comment by ReturnToZender** [comment]
> Small note, the Blueshield ID does not show up on a silver ID card for some reason. Gonna make an update soon to fix that.

Finished testing. It works on a gold ID card, but not a silver ID card. Wonder why?

**Comment by ReturnToZender** [comment]
Figured it out. It's because of the extra captain access. It now has that by default.

File: modular_skyrat/modules/blueshield/code/blueshield.dm [hunk 1]
```diff
@@ -64,7 +63,8 @@
 	box = /obj/item/storage/box/survival/security
 	belt = /obj/item/modular_computer/pda/security
 
-	id_trim = /datum/id_trim/job/blueshield
+	id = /obj/item/card/id/advanced/silver //BUBBER EDIT original: obj/item/card/id/advanced/CENTCOM
+	id_trim = /datum/id_trim/job/blueshield //BUBBER EDIT
```
Comment (Cyprex) [inline modular_skyrat/modules/blueshield/code/blueshield.dm:67]: What trim were they previously using?

**Review by Cyprex — APPROVED** [review]

File: modular_skyrat/modules/blueshield/code/blueshield.dm (hunk 1 above)
Comment (ReturnToZender) [reply modular_skyrat/modules/blueshield/code/blueshield.dm:67]: You know, now that I think about it, it's not a bubber edit. It's the same id_trim.

File: modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm [hunk 2]
```diff
@@ -128,7 +128,7 @@
 
 /obj/structure/closet/secure_closet/nanotrasen_consultant/station
 	name = "\proper nanotrasen consultant's locker"
-	req_access = list(ACCESS_CAPTAIN, ACCESS_CENT_GENERAL)
+	req_access = list("representative") //BUBBER EDIT
```
Comment (Cyprex) [inline modular_skyrat/modules/nanotrasen_rep/code/nanotrasen_consultant.dm:131]:
```suggestion
	req_access = list(ACCESS_NTREP) //BUBBER EDIT
```

**Pull request merged**